src/pages/map/index.tsx

```diff
@@ -83,3 +104,19 @@ const LandmarkCard = ({ landmark }: { landmark: Landmark })  => {
       </CardBody>
   </Card>; 
 };
+
+const LandmarkDrawer = ({ isOpen, onClose, landmark }: { 
```

Comment:
We will add a lot more functions into the drawer. So separate into a different file.